VirtoCommerce.InventoryModule.Web/Scripts/inventory.js

```diff
@@ -7,21 +7,28 @@ if (AppDependencies != undefined) {
 
 angular.module(moduleName, [])
     .run(
-    ['$rootScope', 'platformWebApp.mainMenuService', 'platformWebApp.widgetService', 'platformWebApp.authService', 'platformWebApp.metaFormsService', function ($rootScope, mainMenuService, widgetService, authService, metaFormsService) {
+        ['$rootScope', 'platformWebApp.mainMenuService', 'platformWebApp.widgetService', 'platformWebApp.authService', 'platformWebApp.metaFormsService', function ($rootScope, mainMenuService, widgetService, authService, metaFormsService) {
 
-        //Register widgets in catalog item details
-        widgetService.registerWidget({
-            isVisible: function (blade) { return blade.productType !== 'Digital' && authService.checkPermission('inventory:update'); },
-            controller: 'virtoCommerce.inventoryModule.inventoryWidgetController',
-            template: 'Modules/$(VirtoCommerce.Inventory)/Scripts/widgets/inventoryWidget.tpl.html'
-        }, 'itemDetail');
+            //Register widgets in catalog item details
+            widgetService.registerWidget({
+                isVisible: function (blade) { return blade.productType !== 'Digital' && authService.checkPermission('inventory:update'); },
+                controller: 'virtoCommerce.inventoryModule.inventoryWidgetController',
+                template: 'Modules/$(VirtoCommerce.Inventory)/Scripts/widgets/inventoryWidget.tpl.html'
+            }, 'itemDetail');
 
-        widgetService.registerWidget({    
-            size: [2, 1],
-            controller: 'virtoCommerce.inventoryModule.fulfillmentAddressesWidgetController',
-            template: 'Modules/$(VirtoCommerce.Inventory)/Scripts/widgets/fulfillmentAddressesWidget.tpl.html'
-        }, 'fulfillmentCenterDetail');
+            widgetService.registerWidget({
+                size: [2, 1],
+                controller: 'virtoCommerce.inventoryModule.fulfillmentAddressesWidgetController',
+                template: 'Modules/$(VirtoCommerce.Inventory)/Scripts/widgets/fulfillmentAddressesWidget.tpl.html'
+            }, 'fulfillmentCenterDetail');
 
-        metaFormsService.registerMetaFields('inventoryDetails', []);
-        metaFormsService.registerMetaFields('fulfillmentCenterDetails', []);
-    }]);
+
+            widgetService.registerWidget({
```

Comment:
In case we add blade here, probably we need to remove blade and [its registration](https://github.com/VirtoCommerce/vc-module-core/blob/025ae6e5722f49033899122994c9a65aad22d5fe/VirtoCommerce.CoreModule.Web/Scripts/Fulfillment/fulfillment.js) in vc-module-core in another PR to the same issue.
Now we get this:
![image](https://user-images.githubusercontent.com/15610352/59593752-f6098e80-910b-11e9-8006-f70aa75dbf1b.png)

Reply:
Found https://github.com/VirtoCommerce/vc-module-core/pull/122, so its ok.